terraform/itap-piem.alpha-canada.ca.tf

```diff
@@ -0,0 +1,9 @@
+resource "aws_route53_record" "itap-piem.alpha-canada.ca-A" {
```

Comment:
Terraform resources can't have `.` in the name please replace with underscores or dashes or remove all together

Reply:
My bad, updated.